### DISCUSSION TITLE
Invalid copilot token: missing token: 403

### DISCUSSION THREAD

**Discussion by abhishekHegde2000** [community/community#82765] [Copilot Conversations] [answered]
[INFO] [auth] [2023-12-28T07:11:50.729Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-12-28T07:11:50.732Z] Extension activation failed: "Contact Support. You are currently logged in as abhishekHegde2000."
[INFO] [auth] [2023-12-28T07:12:13.533Z] Invalid copilot token: missing token: 403 

![image](https://github.com/community/community/assets/88794340/19c2f67b-3270-4d92-998b-20b44ee448be)

i am having github copilot access till yesterday. but suddenly i am getting this error while using it.
please help me solve this issue.
all of the copilot extensions are give this error

**Comment by kclinden** [comment]
Same issue here.

**Comment by davecheney** [comment, marked as answer]
@abhishekHegde2000 @kclinden 
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Reply by imtrinity94** [reply]
Tried but still the same.
